ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/AbstractBlockProcessor.java

```diff
@@ -108,7 +108,10 @@ private BlockAwareOperationTracer getBlockImportTracer(
           Optional.ofNullable(protocolContext.getPluginServiceManager())
               .flatMap(serviceManager -> serviceManager.getService(BlockImportTracerProvider.class))
               // if block import tracer provider is not specified by plugin, default to no tracing
-              .orElse((__) -> BlockAwareOperationTracer.NO_TRACING);
+              .orElse((__) -> {
+                LOG.info("Block Import uses NO_TRACING");
```

Comment:
This will be logged every time that a block is processed. Maybe move the log outside of the lambda body or change it to `trace`